packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1766,6 +1766,8 @@ export class AccountHandler {
             sessionToken.id,
             accountResetToken.verificationMethod
           );
+        } else if (recoveryKeyId) {
+          await this.db.verifyTokensWithMethod(sessionToken.id, 'totp-2fa');
```

Comment:
I'd put a comment here about what this is doing. Without understanding the context around this change, it might seem like a mistake at first glance.